integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/builder/DatabricksEnvironmentFacetBuilder.java

```diff
@@ -88,59 +90,83 @@ private Map<String, Object> getDatabricksEnvironmentalAttributes(SparkListenerJo
      * storage. However, that dbutils object is not available inside a spark listener. We must
      * access it via reflection.
      */
-    try {
-      Optional<DbfsUtils> dbfsUtils = getDbfsUtils();
-      if (!dbfsUtils.isPresent()) {
-        dbProperties.put("mountPoints", new ArrayList<DatabricksMountpoint>());
-      } else {
-        dbProperties.put("mountPoints", getDatabricksMountpoints(dbfsUtils.get()));
-      }
+    dbProperties.put("mountPoints", getDatabricksMountpoints());
 
-    } catch (Exception e) {
-      log.warn("Failed to load dbutils in OpenLineageListener:", e);
-      dbProperties.put("mountPoints", new ArrayList<DatabricksMountpoint>());
-    }
     return dbProperties;
   }
 
   // Starting in Databricks Runtime 11, there is a new constructor for DbFsUtils
   // If running on an older version, the constructor has no parameters.
   // If running on DBR 11 or above, you need to specify whether you allow mount operations (true or
   // false)
-  private static Optional<DbfsUtils> getDbfsUtils()
-      throws ClassNotFoundException, InstantiationException, IllegalAccessException,
-          IllegalArgumentException, InvocationTargetException {
-    Class dbutilsClass = Class.forName("com.databricks.dbutils_v1.impl.DbfsUtilsImpl");
+  private static List<DatabricksMountpoint> getDatabricksMountpoints() {
+    Class dbutilsClass;
+    try {
+      dbutilsClass = Class.forName("com.databricks.dbutils_v1.impl.DbfsUtilsImpl");
```

Comment:
More reflection 😞 
I understand that it's needed though.

Reply:
yes